wiki/Skinning/Interface/en.md

```diff
@@ -3,93 +3,118 @@
 
 # Interface
 
-_See also: [Skinning Interface Tutorial](/wiki/Skinning_Interface_Tutorial) and [Interface](/wiki/Interface)_
+*Main page: [Skinning](/wiki/Skinning).*
 
-Interface skinning elements are used in multiple game modes or parts of the osu!client's user interface.
+*See also: [Skinning Interface Tutorial](/wiki/Skinning_Interface_Tutorial) and [Interface](/wiki/Interface)*
 
-## Main Menu
+Interface skinning elements are used in multiple game modes or parts of the osu! client's user interface.
 
-![](img/menu-background.jpg "menu-background.jpg")
+## Main menu
 
-| Animatable   | Beatmap Skinnable |
-|:------------:|:-----------------:|
-| ![No][false] | ![No][false]      |
+`menu-background.jpg`
 
-**Notes:**
+![](img/menu-background.jpg)
 
-- [osu!supporter](/wiki/osu!supporter) required
-- This element is used as the playfield if map does not have a background.
-- This element may use the `.jpg` format.
+| Versions | Animatable | Beatmap Skinnable | Blend Mode | Origin | Suggested SD Size |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| All | ![No][false] | ![No][false] | Normal | Centre | 1366x768 |
+
+Notes:
+
+- [osu!supporter](/wiki/osu!supporter) required.
+- By default, osu! has 5 background images that it will cycle through.
+  - If this element is skinned and the user has osu!supporter tag, this element will override this behaviour.
+- This element is used as the playfield if beatmap does not have a background.
+- This element uses the `.jpg` extension.
+  - `.png` images can be used, however, they must end with `.jpg` to work.
 
 ---
 
-![](img/welcome_text.png "welcome_text.png")
+`welcome_text.png`
+
+![](img/welcome_text.png)
 
-| Animatable   | Beatmap Skinnable |
-|:------------:|:-----------------:|
-| ![No][false] | ![No][false]      |
+| Versions | Animatable | Beatmap Skinnable | Blend Mode | Origin | Suggested SD Size |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| All | ![No][false] | ![No][false] | Normal | Centre | - |
 
-**Notes:**
+Notes:
 
-- [osu!supporter](/wiki/osu!supporter) required
-- This should say "Welcome".
+- [osu!supporter](/wiki/osu!supporter) required.
+- This element appears upon starting the osu! client.
+- This element folds out and expands, then fades out.
 
 ---
 
-_`menu-snow.png`_ (transparent; not included; see small game mode icons)
+`menu-snow.png`
 
-| Animatable   | Beatmap Skinnable |
-|:------------:|:-----------------:|
-| ![No][false] | ![No][false]      |
+| Versions | Animatable | Beatmap Skinnable | Blend Mode | Origin | Suggested SD Size |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| All | ![No][false] | ![No][false] | Additive | Centre | 32x32 |
 
-**Notes:**
+Notes:
 
 - If not skinned, the current game mode's small icon will be used instead.
-- Togglable in the [options](/wiki/options).
-  - This option may forcibly be turned on during winter.
+- Must be enabled in the [options](/wiki/options) to see.
+  - This option may forcibly be enabled during winter.
```

Comment:
Change to `This option may forcibly be enabled during the holidays (christmas)`

Winter means the whole season which is not the case.